BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/csv_file/csv_file.rb

```diff
@@ -37,11 +37,11 @@ def self.file_name_pattern #:nodoc:
     /\.csv\z/i
   end
 
-  def is_viewable? #:nodoc:
-    return false unless self.size.presence
-    return false unless self.size < 400_000  # smaller than the limit in TextFile
-    return false unless is_locally_synced?
-    true
+  def csv_is_viewable? #:nodoc:
+    errors = []
```

Comment:
This is very confusing for developers. there is ALREADY a method 'errors' and 'errors=' on the userfile model that has nothing to do with the array here. part of the ActiveModel::Errors framework, and it may seem these statement are trying to use that API. Rename the variable 'error_messages' at least.